app/views/devise/shared/_error_messages.html.slim

```diff
@@ -0,0 +1,11 @@
+.form__message
```

Comment:
Why does this component need to be dependent on the parent component `form`? Couldn't this component be independent e.g `feedback-message`, or just `message` (while it should be better to avoid single word class names as per our conventions)